tests/std/tests/P0009R18_mdspan_layout_stride_death/test.cpp

```diff
@@ -27,13 +27,20 @@ void test_construction_from_strided_layout_mapping() {
     [[maybe_unused]] layout_stride::mapping<dextents<unsigned char, 1>> m2{m1};
 }
 
+void test_call_operator() {
+    layout_stride::mapping<extents<unsigned, 5, 3, 4>> m;

```

Comment:
Nitpick for followup: We conventionally say `unsigned int` instead of `unsigned`.